src/components/state-badge.component.js

```diff
@@ -13,28 +13,32 @@ type Props = {
   locale: string,
 };
 
-const styles = StyleSheet.create({
-  badge: {
-    padding: 12,
-    paddingTop: 3,
-    paddingBottom: 3,
-    borderRadius: 20,
-  },
-  mergedIssue: {
-    backgroundColor: colors.purple,
-  },
-  openIssue: {
-    backgroundColor: colors.green,
-  },
-  closedIssue: {
-    backgroundColor: colors.red,
-  },
-  text: {
-    fontSize: normalize(12),
-    ...fonts.fontPrimarySemiBold,
-    color: colors.white,
-  },
-});
+const Badge = styled.View`
+  padding: 12;
+  padding-top: 3;
+  padding-bottom: 3;
+  border-radius: 20;
+  background-color: ${({ issueState }) => {
+    switch (issueState) {
+      case 'merged':
+        return colors.purple;
+      case 'open':
+        return colors.green;
+      case 'closed':
+        return colors.red;
+      default:
+        break;
```

Comment:
You can return `colors.black` here instead of returning it on line 33.

Reply:
completed!